### PR TITLE
feat(onboarding): add alpha onboarding page

### PR DESCRIPTION
**Pull request by JonasBa**
CleanShot 2022-06-28 at 16 40 34@2x

Adds a simple alpha onboarding page w/o the showing/hiding mechanism (will be added in followup pr)

### CODE REVIEW

File: static/app/views/profiling/profilingOnboarding.tsx [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+            <Alert>
+              {t(
+                `Profiling is only possible with sentry-cocoa and sentry-java SDKs right now. We don’t support React Native or Flutter yet.`
+              )}
+            </Alert>
+
+            <ProfilingSteps>
+              <li>
+                {t(
+                  'Make sure your SDKs are upgraded to 6.0.0(sentry-java) or 7.13.0 (sentry-cocoa).'
```
Comment (Zylphrex) [inline static/app/views/profiling/profilingOnboarding.tsx:41]:
```suggestion
                  'Make sure your SDKs are upgraded to 6.0.0 (sentry-java) or 7.13.0 (sentry-cocoa).'
```

File: static/app/views/profiling/profilingOnboarding.tsx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+                {t(
+                  `Setup performance transactions in your app if you haven’t already → `
+                )}{' '}
+                <a
+                  href="https://docs.sentry.io/product/performance/"
+                  target="_blank"
+                  rel="noreferrer"
+                >
+                  {t('https://docs.sentry.io/product/performance/')}
+                </a>
```
Comment (Zylphrex) [inline static/app/views/profiling/profilingOnboarding.tsx:54]: This should use an `ExternalLink` component.

File: static/app/views/profiling/profilingOnboarding.tsx [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+                {t(
+                  'or email us at profiling@sentry.io with any questions or if you need help setting it all up! There’s also a page with some more details and a troubleshooting section at'
+                )}{' '}
+                <a
+                  href="https://sentry.notion.site/Profiling-Beta-Testing-Instructions-413ecdd9fcb34b3a8b57806280bf2ecb"
+                  target="_blank"
+                  rel="noreferrer"
+                >
+                  our notion page
+                </a>
```
Comment (Zylphrex) [inline static/app/views/profiling/profilingOnboarding.tsx:88]: - Use `ExternalLink`
- `{t('...')}`

File: static/app/views/profiling/profilingOnboarding.tsx [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+              </li>
+
+              <Actions>
+                <Button priority="primary">I'm done</Button>
+                <Button>Dismiss</Button>
```
Comment (Zylphrex) [inline static/app/views/profiling/profilingOnboarding.tsx:93]:
```suggestion
                <Button priority="primary">{t("I'm done")}</Button>
                <Button>{t('Dismiss')}</Button>
```

**Comment by JonasBa** [comment]
For sure @jernejstrasner 

localhost_7999_

File: static/app/views/profiling/profilingOnboarding.tsx [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+            </p>
+
+            <Alert>
+              {t(
+                `Profiling is only possible with sentry-cocoa and sentry-java SDKs right now. We don’t support React Native or Flutter yet.`
+              )}
+            </Alert>
+
+            <ProfilingSteps>
+              <li>
```
Comment (evanpurkhiser) [inline static/app/views/profiling/profilingOnboarding.tsx:52]: I think we have a list component. Something we could use here?

**Review by Zylphrex — APPROVED** [review]

**Pull request merged**